jangjuseong/controllers/userController.js

```diff
@@ -0,0 +1,26 @@
+const userService = require('../services/userService');
+
+const signUp = async (req, res) => {
+  const { name, email, password, profileImage } = req.body;
```

Comment:
회원가입을 할 때 반드시 필요한 데이터가 있을 것이라고 생각이 듭니다.

해당 부분에서도 반드시 필요한 데이터가 요청으로 입력되지 않았을 때 예외처리를 해주면 좋을 것 같습니다!

```suggestion
  const { name, email, password, profileImage } = req.body;
  
  if (!name | !email | !password ) {
    throw new Error("Key Error!");
  }
```